forum/forum/urls.py

```diff
@@ -41,6 +42,12 @@
     path('api/token/verify/', TokenVerifyView.as_view(), name='token_verify'),  
 
     # Authentication URLs (using Djoser)
-    path('api/v1/auth/', include('djoser.urls')),       
-    path('api/v1/auth/', include('djoser.urls.jwt')),     
+    path('api/v1/auth/', include('djoser.urls')),
+    path('api/v1/auth/', include('djoser.urls.jwt')),
+
+    path('reset_password/', auth_views.PasswordResetView.as_view(), name ='reset_password'),
+    path('reset_password_sent/', auth_views.PasswordResetDoneView.as_view(), name ='password_reset_done'),
+    path('reset/<uidb64>/<token>', auth_views.PasswordResetConfirmView.as_view(), name ='password_reset_confirm'),
+    path('reset_password_complete/', auth_views.PasswordResetCompleteView.as_view(), name ='password_reset_complete'),
+
```

Comment:
Since you are already using Djoser for authentication, you may want to ensure all your authentication-related URLs (Djoser and password reset views) are within the same namespace or group. This keeps the code cleaner and easier to maintain.
You could move the password reset views under the `api/v1/auth/` namespace like this:
```
path('api/v1/auth/reset_password/', auth_views.PasswordResetView.as_view(), name='reset_password'),
path('api/v1/auth/reset_password_sent/', auth_views.PasswordResetDoneView.as_view(), name='password_reset_done'),
path('api/v1/auth/reset/<uidb64>/<token>/', auth_views.PasswordResetConfirmView.as_view(), name='password_reset_confirm'),
path('api/v1/auth/reset_password_complete/', auth_views.PasswordResetCompleteView.as_view(), name='password_reset_complete'),
```